docs/source/models/causes/other_causes/index.rst

```diff
@@ -82,29 +70,23 @@ States Data
      - 
    * - C
      - excess mortality rate
-     - :math:`CSMR_\text{other causes}` 
-     - CSMR is equal to the EMR because prevalence is equal to 1
+     - 0
+     - 
    * - C
      - disabilty weight
-     - :math:`YLDs_{c294} - \sum_{cid}^{n} YLDs_{cid}`
-     - Where :math:`\sum_{cid}^{n} YLDs_{cid}` is the sum of YLDs due to all specified modeled components (may be causes, sequelae, and/or impairments). List of such IDs must be provided in concept model document.
-   * - ALL
-     - cause specific mortality rate
-     - :math:`ACMR - \sum_{cid}^{n} CSMR_{cid}`
-     - Where :math:`\sum_{cid}^{n} CSMR_{cid}` is the sum of the CSMRs of modeled causes. List of modeled cause IDs must be provided in concept model document.
+     - :math:`YLDs_{c294} - YLDs_{r192} - (YLDs_{c366} - YLDs_{s182,s183,s184}) + \frac{YLD_{c366} - YLDs_{\text{s182,s183,s184}}}{\text{preg_rate}}`
```

Comment:
```suggestion
     - :math:`YLDs_{c294} - YLDs_{r192} - (YLDs_{c366} - YLDs_{\text{s182,s183,s184}}) * (1 + \frac{1}{\text{preg_rate}})`
```

Note: I'm just mechanically simplifying here. I don't actually understand this equation. I would have guessed that the equation would be this:

```
YLDs_{c294} - YLDs_{r192} - (YLDs_{c366} - YLDs_{\text{s182,s183,s184}}) / preg_rate
```

Reply:
I'll break down what's happening here: 
- YLDs_{c294} # start with population all cause YLD rate that we assume to generalize to the pregnant population
- - YLDs_{r192} # subtract anemia YLDs that we assume generalizes to the pregnant population
- - (YLDs_{c366} - YLDs_{\text{s182,s183,s184}}) # subtract off _population level_ maternal disorders YLD rate excluding maternal disorders YLD sequelae that overlap with anemia (we don't want to subtract off these sequelae twice!). We're subtracting the population level maternal disorders YLD rate and will add back the pregnancy-specific maternal disorder YLD rate in the next step
- + \frac{YLD_{c366} - YLDs_{\text{s182,s183,s184}}}{\text{preg_rate}} # now we are adding back in the maternal disorder YLD rate, but adjusted up to be specific to the pregnant population by dividing by the incidence rate of pregnancy 


Does this clarify what's happening here?

Reply:
Basically s182,s183,s184 are anemia due to maternal hemorrhage, so these sequelae are included in both YLDs due to anemia impairment and YLDs due to the maternal disorders cause. We decided that we will count them as part of the anemia impairment rather than the maternal disorders cause, so we always subtract them off of YLDs_c366

Reply:
Thanks! Your update makes this much more clear 👍